installing/installing_oci/installing-oci-assisted-installer.adoc

```diff
@@ -1,38 +1,41 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="installing-oci-assisted-installer"]
-= Using the Assisted Installer to install a cluster on OCI
+= Installing a cluster on {oci-first-no-rt} by using the Assisted Installer
 include::_attributes/common-attributes.adoc[]
 :context: installing-oci-assisted-installer
 
 toc::[]
 
-From {product-title} {product-version} and later versions, you can use the Assisted Installer to install a cluster on {oci-first} by using infrastructure that you provide.
+From {product-title} {product-version} and later versions, you can use the {ai-full} to install a cluster on {oci-first} by using infrastructure that you provide.
 
 // The Assisted Installer and OCI overview
 include::modules/installing-oci-about-assisted-installer.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
-* link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/2024[Assisted Installer for {product-title}]
+* link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/2024[{ai-full} for {product-title}]
```

Comment:
I recommend removing the version (2024) so that the link does not have to be updated when the version changes in 2025.

Reply:
Thanks. For a follow-up PR. I need to address merge conflicts on 4.14-4.16